libtiledbsoma/src/pyapi/libtiledbsoma.cc

```diff
@@ -205,26 +198,13 @@ PYBIND11_MODULE(libtiledbsoma, m) {
                     array.attr("_export_to_c")(
                         arrow_array_ptr, arrow_schema_ptr);
 
-                    if (!strcmp(arrow_schema.format, "L") ||
-                        !strcmp(arrow_schema.format, "l")) {
-                        std::span<uint64_t> data{
-                            (uint64_t*)arrow_array.buffers[1],
+                    // 'l' = arrow data type int64
```

Comment:
thank you for this comment ❤️

Reply:
For reference, see [Arrow C inteface format strings](https://arrow.apache.org/docs/format/CDataInterface.html#data-type-description-format-strings)